django_mongodb/fields/objectid.py

```diff
@@ -0,0 +1,27 @@
+from bson import ObjectId
+from bson.errors import InvalidId
+from django.core import exceptions
+from django.db.models.fields import Field
+
+from .objectid_mixin import ObjectIdMixin
```

Comment:
It's okay to put ObjectIdMixin in this file rather than a separate one.

Reply:
auto also uses it, shall I move auto to objectid?